composer.json

```diff
@@ -25,14 +25,34 @@
         "ockcyp/covers-validator": "1.3.3",
         "phpunit/phpunit": "^8.5.13||^9.5.0",
         "php-parallel-lint/php-console-highlighter": "0.5",
-        "php-parallel-lint/php-parallel-lint": "1.3.1"
+        "php-parallel-lint/php-parallel-lint": "1.2.0",
+        "friendsofphp/php-cs-fixer": "^3.5",
+        "brainmaestro/composer-git-hooks": "^2.8"
     },
     "scripts": {
+        "post-install-cmd": "cghooks add --ignore-lock",
+        "post-update-cmd": "cghooks update",
         "test": [
             "parallel-lint . --exclude vendor",
             "covers-validator",
             "phpunit --coverage-text"
         ],
-        "cover": "phpunit --coverage-html coverage"
+        "cover": "phpunit --coverage-html coverage",
+        "cghooks": "vendor/bin/cghooks",
+        "fix": "php-cs-fixer fix --config=.php-cs-fixer.php"
+    },
+    "extra": {
+        "hooks": {
+            "config": {
+                "stop-on-failure": ["pre-commit"]
+            },
+            "pre-commit": [
+                "composer run-script fix",
```

Comment:
Does `@fix` work here?

Inside scripts itself you can do something like the below to call the other scripts with shorthand (saving duplicating commands). Curious if this works here too

```
	"scripts": {
		"lint": "parallel-lint --exclude node_modules --exclude vendor",
		"phpcs": "phpcs -p -s --cache",
		"test": [
			"@lint .",
			"@phpcs ."
		]
	}
```